api/channel.go

```diff
@@ -1142,3 +1146,12 @@ func updateNotifyProps(c *Context, w http.ResponseWriter, r *http.Request) {
 	}
 
 }
+
+func GetChannelMember(channelId, userId string) (model.ChannelMember, *model.AppError) {
+	result := <-Srv.Store.Channel().GetMember(channelId, userId)
+	if result.Err != nil {
+		return model.ChannelMember{}, result.Err
```

Comment:
It's not `return nil, result.Err` as `Channel().GetMember` does not return a pointer. I can manually cast it to a pointer. I'm not sure if I should stick to how the store was defined (not a pointer), or just make everything a pointer. 

I can replace the other calls to GetMember with this, it'll just involve some refactoring.